android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  package="com.quillraven.gamename" >
+
+  <application
+    android:icon="@drawable/ic_launcher"
+    android:isGame="true"
+    android:label="@string/app_name"
+    android:theme="@style/GdxTheme"
+    tools:ignore="AllowBackup,GoogleAppIndexingWarning">
+    <activity
+      android:name="com.game.quillyjumper.android.Launcher"
+      android:label="@string/app_name"
+      android:screenOrientation="landscape"
+      android:configChanges="keyboard|keyboardHidden|navigation|orientation|screenSize|screenLayout">
+      <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+        <category android:name="android.intent.category.LAUNCHER" />
+      </intent-filter>
+    </activity>
+  </application>
+
+</manifest>
```

Comment:
Seems like file got moved from `android` to `android source folder`. Is this really ok?

Reply:
That's how the flutter generator and flutter is by Google so I trust it
Also some example projects did it like this